specifyweb/frontend/js_src/lib/localization/preferences.ts

```diff
@@ -296,6 +296,9 @@ export const preferencesText = createDictionary({
     'fr-fr': 'Apparence',
     'uk-ua': 'Зовнішній вигляд',
   },
+  buttonColor: {
```

Comment:
Change the key from `buttonColor` to `buttons` - try to have the language key match the localized English string. Exceptions:
- Localized string is long/has special characters (in this case key should be summarizing it)
- The string is a description for another string/used as a tooltip text. I.e, `searchAlgorithm` and `searchAlgorithmDescription` - this helps translators see the relationship between the button/heading and the accompanying tooltip

More details: https://github.com/specify/specify7/tree/production/specifyweb/frontend/js_src/lib/localization#front-end-localization